system/tests/test_expander.py

```diff
@@ -36,7 +36,7 @@ def test_wildcards(self):
         assert pipe_sequence.lst[0].args[0] == 'README.md'
 
         pipe_sequence = self._get_pipe_sequence(r'ls *stash*')
-        assert 'stash.py' in pipe_sequence.lst[0].args
+        # assert 'stash.py' in pipe_sequence.lst[0].args
```

Comment:
Why remove this assertion?

Reply:
This test attempts to expand `*stash*` to `stash.py`, `getstash.py` and `launch_stash.py`.
`stash.py` was renamed to `core.py` to fix issues with the `import` mechanic changes in py3, so `*stash*` can no longer be completed to `stash.py`.
Since there is no reason to replace this assertion in this test, i removed it.

Reply:
Would it be appropriate to have __assert 'core.py' in pipe_sequence.lst[0].args__

Reply:
@cclauss no. We test for expansion of `*stash*`, which can never be expanded to `core.py`.
An alternative would be adding a new test which checks for completion of `core*`, but since we dont test for the existence of that file but for completion, there is no reason to do so.